web/src/components/molecules/Asset/AssetList/AssetListHeader/index.tsx

```diff
@@ -1,27 +1,45 @@
 import styled from "@emotion/styled";
 
-import { UploadChangeParam, UploadFile } from "@reearth-cms/components/atoms/Upload";
+import { UploadProps, UploadFile } from "@reearth-cms/components/atoms/Upload";
 import UploadAsset from "@reearth-cms/components/molecules/Asset/UploadAsset";
 
 type Props = {
   title: string;
   subTitle: string;
-  handleUpload: (info: UploadChangeParam<UploadFile<any>>) => void;
+  fileList: UploadFile<any>[];
```

Comment:
Can `UploadFile` actually be anything? Don't we need to restrict what file types are allowed to be uploaded and then set those allowed types here?

Reply:
replaced with UploadFile<File>[]